src/scripts.js

```diff
@@ -1,11 +1,90 @@
-// This is the JavaScript entry file - your code begins here
-// Do not delete or rename this file ********
-
-// An example of how you tell webpack to use a CSS (SCSS) file
 import './css/styles.css';
-
+import UserRepository from './UserRepository';
+import Destinations from './Destinations'
+import User from './User';
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
 
+//Global Variables
+let userRepo;
+let destinations;
+let trips;
+
+//Event Listeners
+window.addEventListener('load', function(){
+    resolvePromises()
+})
+// API Calls GET Requests
+function loadTravelerData(){
+    const travelersURL = 'http://localhost:3001/api/v1/travelers'
+    return fetch(travelersURL)
+    .then((response) => {
+        if (response.ok){
+                return response.json()
+        }
+    })
+    .then((data) => {
+        console.log(data)
+        return data
+    })
+}
+
+function loadDestinationData(){
+    const destinationURL = 'http://localhost:3001/api/v1/destinations'
+    return fetch(destinationURL)
+    .then((response) => {
+        if (response.ok){
+                return response.json()
+        }
+    })
+    .then((data) => {
+        console.log(data)
+        return data
+    })
+}
+
+function loadTripsData(){
+    const tripsURL = 'http://localhost:3001/api/v1/trips'
+    return fetch(tripsURL)
+    .then((response) => {
+        if (response.ok){
+                return response.json()
+        }
+    })
+    .then((data) => {
+        console.log(data)
+        return data
+    })
+}
+
```

Comment:
If you feel like it/ have the time. I'd refactor these three fetch functions into a single function where you can pass a string into it to complete the url depending on what you want it to fetch.  It would DRY up this part quite a bit.